tests/test_rooms/test_room_crud_with_tags.py

```diff
@@ -66,3 +67,12 @@ def test_query_room_by_id_with_tags(self):
             self,
             room_query_by_id,
             room_query_by_id_response)
+
+    def test_create_room_with_invalid_locationId(self):
+        """
+        Testing for room creation with invakid location id
```

Comment:
Nice work. However, you have a typo here.

Reply:
Thanks @caveinn, worked on this.